apps/vr-tests-react-components/src/stories/Tag/InteractionTag.stories.tsx

```diff
@@ -16,68 +16,91 @@ export default {
   decorators: [story => withStoryWrightSteps({ story, steps })],
 } as ComponentMeta<typeof InteractionTag>;
 
-export const Default = () => <InteractionTag>Primary Text</InteractionTag>;
+export const Default = () => (
+  <InteractionTag>
+    <Primary>Primary Text</Primary>
```

Comment:
`Primary` and `Secondary` are not great names IMO. They can have naming clashes since export everything from react-components.

Also there has been some discussion in our tech syncs recently that we should make sub components more identifiable to their parent components

Reply:
Let's try and figure out better names for these JSX elements

Reply:
Yeah I was in discussion with designer on this, we're looking for some existing examples that are similar but we don't have anything good so far :(

Reply:
Okay we landed on `InteractionTagPrimary` and `InteractionTagSecondary`. Even though this is long, but it's probably the most accurate.